src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -854,7 +854,7 @@ public async Task SendPingToExternalHostWithLowTtlTest()
             options.Ttl = 1;
             // This should always fail unless host is one IP hop away.
             pingReply = await ping.SendPingAsync(host, TestSettings.PingTimeout, TestSettings.PayloadAsBytesShort, options);
-            Assert.True(pingReply.Status == IPStatus.TimeExceeded || pingReply.Status == IPStatus.TtlExpired);
+            Assert.True(pingReply.Status == IPStatus.TtlExpired);
```

Comment:
I worry this is going to be problematic. We _should_ get `IPStatus.TtlExpired` _if_ we get ICMP back. However, depending on the network setup and gear there may not be answer and the ping will fail with plain `Timeout`. 

But I don't know hoe to make it both precise and reliable without ownership of the infrastructure.

Reply:
To be on the safe side, I reverted this.

Reply:
I think we need some owner of the infrastructure